### DISCUSSION TITLE
[Bug]:关于本地远程桌面因分辨率问题无法实现自动化

### DISCUSSION THREAD

**Discussion by tatookan** [moesnow/March7thAssistant#152] [Q&A]
### 请确保您已阅读以上注意事项，并勾选下方的确认框。

- [X] 我已经使用一个干净且无其它非必要的插件的环境测试过，问题依旧存在。
- [X] 我已经在 [Issue Tracker](https://github.com/moesnow/March7thAssistant/issues) 中找过我要提出的问题，没有找到相同问题的ISSUE。
- [X] 我已知晓并同意，若这个 Issue 是关于其他非程序本身问题，则我的 Issue 可能会被无条件自动关闭/锁定。
- [X] 我已知晓并同意，如果我不按照模板提供的指示进行填写，则我的 Issue 可能会被无条件删除。

### March7thAssistant 版本 （或 Commit Id）

实时同步的最新版本

### 重现步骤

我的显示配置:1070ti、 显示器2K分辨率
本地远程桌面成功部署并进入游戏，但是游戏无法修改分辨率为1920*1080.
我尝试了以下方法均没有得到解决
1、尝试通过英伟达显卡控制器修改分辨率
2、尝试修改游戏注册表

### 期望的结果是什么？

兼容更多的分辨率或解决本地远程桌面调整分辨率的问题

### 实际的结果是什么？

兼容更多的分辨率或解决本地远程桌面调整分辨率的问题

**Comment by moesnow** [comment]
目前支持 1920*1080 及以上 16:9 的分辨率，本地远程桌面分辨率的问题不应该由小助手来处理。

有尝试过新版本自带的“自动修改分辨率”功能吗？

**Reply by rosebudgrave** [reply]
自动修改分辨率不生效,我是使用小助手启动的
![1712246550833](https://github.com/moesnow/March7thAssistant/assets/45752666/be7ad8ce-124b-448e-b2ff-c2f22125665b)
![1712246600018](https://github.com/moesnow/March7thAssistant/assets/45752666/2fa192fd-ffa0-461d-8f4e-39ca7adc93b1)

**Reply by rosebudgrave** [reply]
我也是使用的多用户远程桌面,这个分辨率没有1920*1080
